pnut.c

```diff
@@ -2011,11 +2011,52 @@ void get_tok() {
 }
 
 // parser
-
 #if defined DEBUG_CPP || defined DEBUG_EXPAND_INCLUDES || defined NICE_ERR_MSG
 #include "debug.c"
 #endif
 
+
+#define ANSI_RED     "\x1b[31m"
+#define ANSI_GREEN   "\x1b[32m"
+#define ANSI_YELLOW  "\x1b[33m"
+#define ANSI_BLUE    "\x1b[34m"
+#define ANSI_MAGENTA "\x1b[35m"
+#define ANSI_CYAN    "\x1b[36m"
+#define ANSI_RESET   "\x1b[0m"
+
+void parse_error(char * msg, int token) {
+#ifdef NICE_ERR_MSG 
+  //Error header
```

Comment:
Nit: Can we move the `ANSI_*` macros inside this block so they don't get defined when not in use?

Reply:
sure, I thought they might be useful elsewhere so thats why I didn't initially

Reply:
It's mostly because shells get slower as more memory is allocated, so it's important to skip over anything that's unused and that increase the memory usage of pnut.